### ISSUE TITLE
secondary catatact

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#10845]
**Preferred term label:**
secondary catatact

**Synonyms**

**Definition (free text, please give PubMed ID)**
Complication following cataract surgery

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@daboe01 
The HPO typically does not code etiologies.
Would the following be correct

Primary label: Posterior capsule opacification
Synonym: Secondary cataract

Parent term -- this is not actually a cataract, right? So we would probably want to have a parent term that is "Abnormal lens capsule morphology"?

Definition: Posterior capsule opacification, also known as secondary cataract, is the most frequent complication following manual extracapsular cataract extraction. Secondary cataract results from the proliferation and migration of residual epithelial cells in the posterior region of the lens capsule into the visual axis, leading to thickening and reduction in transparency (opacification), which in turn can cause a decrease in visual acuity.

PMID:21750620

**Comment by pnrobinson** [comment]
@daboe01 
Adding new term: 
- ID: HP:6001382
- label: Posterior capsule opacification
- parents: Abnormal lens capsule morphology
- pmid: PMID:21750620
- def: Posterior capsule opacification, also known as secondary cataract, is the most frequent complication following manual extracapsular cataract extraction. Secondary cataract results from the proliferation and migration of residual epithelial cells in the posterior region of the lens capsule into the visual axis, leading to thickening and reduction in transparency (opacification), which in turn can cause a decrease in visual acuity.
- comment: 
- orcid: orcid:0000-0001-9042-0508
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10845
- synonyms: Secondary cataract(NONE)
